mne/_fiff/write.py

```diff
@@ -133,9 +133,8 @@ def write_string(fid, kind, data):
     except UnicodeEncodeError:
         str_data = str(data).encode("latin1", errors="xmlcharrefreplace")
     data_size = len(str_data)  # therefore compute size here
-    my_dtype = ">a"  # py2/3 compatible on writing -- don't ask me why
     if data_size > 0:
-        _write(fid, str_data, kind, data_size, FIFF.FIFFT_STRING, my_dtype)
+        _write(fid, str_data, kind, data_size, FIFF.FIFFT_STRING, "S")
```

Comment:
@mscheltienne I'm hijacking your PR to sneak in some CI fixes, hope you don't mind :)

Reply:
Nothing unusual :)